FlatUI-theme.el

```diff
@@ -33,6 +33,10 @@
 (defvar fui-alizarin "#e74c3c")
 (defvar fui-clouds "#ecf0f1")
 (defvar fui-concrete "#95a5a6")
+(defvar fui-asbestos "#7f8c8d")
```

Comment:
If you look down, those colors are already define, but with dumber names (so I can remember them).